### PR TITLE
fix(timeoutHeight): removing ZeroHeight() and bumping sdk version, fixing rly.sh

### PR DESCRIPTION
**Pull request by seantking**
Bumping the SDK version and using the most recent relayer commands in `rly.sh`. 

This PR depends on https://github.com/cosmos/relayer/pull/483

### CODE REVIEW

File: x/ibc-account/keeper/relay.go [hunk 1]
```diff
@@ -40,7 +40,10 @@ func (k Keeper) TryRegisterIBCAccount(ctx sdk.Context, sourcePort, sourceChannel
 		Data: salt,
 	}
 
-	timeoutHeight := clienttypes.ZeroHeight()
```
Comment (seantking) [inline x/ibc-account/keeper/relay.go:43]: @colin-axner changing this to clienttypes.ZeroHeight() seems to cause the relayer to crash with a nondescript error

Comment (colin-axner) [reply x/ibc-account/keeper/relay.go:43]: This is probably a bug in the relayer. Lets open an issue and fix it

Comment (colin-axner) [reply x/ibc-account/keeper/relay.go:43]: see [relayer issue](https://github.com/cosmos/relayer/issues/481)

Comment (colin-axner) [reply x/ibc-account/keeper/relay.go:43]: It should be noted that hardcoding `revision height` will break for any chain without `-2` as a suffix

Comment (colin-axner) [reply x/ibc-account/keeper/relay.go:43]: otherwise any chain-ids with a revision numebr > 2 will timeout automatically

Comment (seantking) [reply x/ibc-account/keeper/relay.go:43]: Wdyt https://github.com/cosmos/relayer/pull/483

**Review by colin-axner — APPROVED** [review]

**Pull request merged**